api/v1/server/oas/transformers/workflow_run.go

```diff
@@ -544,6 +549,7 @@ func ToScheduledWorkflowsFromSQLC(scheduled *dbsqlc.ListScheduledWorkflowsRow) *
 		WorkflowRunName:      &scheduled.WorkflowRunName.String,
 		Method:               gen.ScheduledWorkflowsMethod(scheduled.Method),
 		Priority:             &scheduled.Priority,
+		Input:                &input,
```

Comment:
we're returning the input on the list query here? we'll likely have the same issues as list runs at some point... your call on how we want to handle this